com.unity.sg2/Tests/Generation/GenerationTests.cs

```diff
@@ -50,8 +50,8 @@ public static void Setup()
             graph.AddNode<TestAddNode>("Add1").SetPortField("In1", "c0", 1f); //(1,0,0,0)
             graph.AddNode<TestAddNode>("Add2").SetPortField("In2", "c1", 1f); //(0,1,0,0)
             graph.AddNode<TestAddNode>("Add3");
-            graph.TryConnect("Add1", "Out", "Add3", "In1", registry);
-            graph.TryConnect("Add2", "Out", "Add3", "In2", registry); //should be (1,1,0,0)
+            graph.TryConnect("Add1", "Out", "Add3", "In1");
```

Comment:
Getting rid of the registry reference here as its not needed anymore, same for all these calls below